app/build.gradle

```diff
@@ -92,4 +92,6 @@ dependencies {
     androidTestImplementation "com.google.dagger:hilt-android-testing:$versions.hilt"
 
     kaptAndroidTest "com.google.dagger:hilt-android-compiler:$versions.hilt"
+
+    implementation "androidx.core:core-splashscreen:1.0.0-alpha01"
```

Comment:
@amit-metri  Please declare version in versions.gradle & reference it here.

Reply:
done